radixdlt/src/test/java/com/radixdlt/sanitytestsuite/model/SanityTestSuiteRoot.java

```diff
@@ -0,0 +1,82 @@
+package com.radixdlt.sanitytestsuite.model;
+
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+// CHECKSTYLE:OFF checkstyle:VisibilityModifier
```

Comment:
Please don't turn off checkstyle warnings. Member fields here should definitely not be public.

Reply:
Changed to final

Reply:
Please delete the checkstyle warning suppression as well.